src/TrajectoryPlanner/src/TrajectoryGenerator.cpp

```diff
@@ -496,6 +497,18 @@ bool TrajectoryGenerator::isTrajectoryAsked()
     return m_generatorState == GeneratorState::Called;
 }
 
+bool TrajectoryGenerator::generateTrajectoriesFromFootprintsStepAdjustment(std::shared_ptr<FootPrint> left, std::shared_ptr<FootPrint> right,
```

Comment:
Since now there is a single trajectory generator object, there is no need to change the name I guess. ``generateTrajectoriesFromFootprints`` is enough I guess.

Reply:
Done!